Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos/ruby-25-centos7
+FROM quay.io/centos7/ruby-25-centos7
```

Comment:
Looks good to me as a quick fix.

FYI there was a suggestion to use UBI image here:
https://github.com/3scale/porta/pull/2476#discussion_r645673284

Perhaps can be done as a separate task as it will likely require more changes that simple image URI.

Reply:
@akostadinov I think @hallelujah already had a go at that [here](https://github.com/3scale/system-builder/pull/22#discussion_r647305629)

If we can make that work - by all means!! - but, yes, let's keep that separate from this PR that we need as the quick fix ;)